infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -161,21 +162,46 @@ public void validatorsInitialised(final List<String> validators) {
     }
   }
 
-  public void validatorsDoppelgangerDetected(final Map<Integer, String> doppelgangersInfo) {
+  public void doppelgangerDetectionStart(Set<String> publicKeys) {
+    log.info("Starting doppelganger detection for public keys: {}", String.join(", ", publicKeys));
+  }
+
+  public void doppelgangerDetectionEnd(Set<String> publicKeys) {
+    log.info(
+        "No validators doppelganger detected after 2 epochs. Stopping doppelganger detection for public keys {}.",
+        String.join(", ", publicKeys));
+  }
+
+  public void doppelgangerCheck(long epoch, long slot, Set<String> publicKeys) {
+    log.info(
+        "Performing doppelganger check. Epoch {}, Slot {}, Public keys {}",
+        epoch,
+        slot,
+        String.join(", ", publicKeys));
+  }
+
+  public void validatorsDoppelgangerDetected(final Map<UInt64, String> doppelgangersInfo) {
     String doppelgangersLogInfo =
         doppelgangersInfo.entrySet().stream()
             .map(
                 doppelgangerInfo ->
                     StringUtils.isBlank(doppelgangerInfo.getValue())
-                        ? String.format("Index: %d", doppelgangerInfo.getKey())
+                        ? String.format("Index: %s", doppelgangerInfo.getKey())
                         : String.format(
-                            "Index: %d, Public key: %s",
+                            "Index: %s, Public key: %s",
                             doppelgangerInfo.getKey(), doppelgangerInfo.getValue()))
             .collect(Collectors.joining("\n", "\n", "\n"));
     log.fatal(
         "Detected {} validators doppelganger: {}", doppelgangersInfo.size(), doppelgangersLogInfo);
   }
 
+  public void doppelgangerDetectionAlert(final Set<String> doppelgangerPublicKeys) {
+    log.fatal(
```

Comment:
This isn't fatal if we're not exiting as a result - it would just be an error level.